.gitignore

```diff
@@ -5,6 +5,7 @@ __pycache__/
 *.vscode
 __pycache__
 *.pyc
+playground.py
```

Comment:
I don't want this in our global gitignore because this is just your naming convention (e.g. for me these files are named `tmp.py`) and if we add them all it would become a bit messy

Reply:
Good catch :D I added it to the wrong ignore file 🙈